> Thanks. Do you know how to solve problem with Shippable?

Well, just click thru the web UI to get to the error messages. E.g.
https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/35722/1/tests shows that not all cases were converted:

~~~
FAILED: ccache /opt/sdk/zephyr-sdk-0.9.5/sysroots/x86_64-pokysdk-linux/usr/bin/arm-zephyr-eabi/arm-zephyr-eabi-gcc -DBUILD_VERSION=v1.14.0-rc1-623-g327632cfc70b -DKERNEL -DSTM32F401xE -DUSE_FULL_LL_DRIVER -DUSE_HAL_DRIVER -D_FORTIFY_SOURCE=2 -D__ZEPHYR__=1 -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/kernel/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include/drivers -Izephyr/include/generated -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/soc/arm/st_stm32/stm32f4 -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/lib/libc/minimal/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/cmsis/Include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/st/stm32cube/stm32f4xx/soc -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/st/stm32cube/stm32f4xx/drivers/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/st/stm32cube/stm32f4xx/drivers/include/Legacy -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/subsys/testsuite/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/subsys/testsuite/ztest/include -isystem /opt/sdk/zephyr-sdk-0.9.5/sysroots/x86_64-pokysdk-linux/usr/lib/arm-zephyr-eabi/gcc/arm-zephyr-eabi/6.2.0/include -isystem /opt/sdk/zephyr-sdk-0.9.5/sysroots/x86_64-pokysdk-linux/usr/lib/arm-zephyr-eabi/gcc/arm-zephyr-eabi/6.2.0/include-fixed -Os -nostdinc -g -Wall -Wformat -Wformat-security -Wno-format-zero-length -imacros /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/sanity-out/96b_carbon/tests/kernel/mem_protect/mem_protect/kernel.memory_protection/zephyr/include/generated/autoconf.h -ffreestanding -Wno-main -fno-common --sysroot /opt/sdk/zephyr-sdk-0.9.5/sysroots/armv5-zephyr-eabi/usr -mthumb -mcpu=cortex-m4 -Werror -Wno-deprecated-declarations -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -Wno-pointer-sign -Wno-unused-but-set-variable -fno-reorder-functions -fno-defer-pop -Werror=implicit-int -Wpointer-arith -ffunction-sections -fdata-sections -mabi=aapcs -march=armv7e-m -std=c99 -MD -MT CMakeFiles/app.dir/src/kobject.c.obj -MF CMakeFiles/app.dir/src/kobject.c.obj.d -o CMakeFiles/app.dir/src/kobject.c.obj   -c /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/tests/kernel/mem_protect/mem_protect/src/kobject.c
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/tests/kernel/mem_protect/mem_protect/src/kobject.c: In function test_kobject_revoke_access:
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/tests/kernel/mem_protect/mem_protect/src/kobject.c:161:24: error: implicit declaration of function MSEC [-Werror=implicit-function-declaration]
  k_sem_take(&sync_sem, MSEC(100));
                        ^~~~
~~~

--- a/include/sys_clock.h
+++ b/include/sys_clock.h
@@ -213,17 +213,6 @@ struct _timeout {
 	_timeout_func_t fn;
 };
 
-/*
- * Number of ticks for x seconds. NOTE: With MSEC() or USEC(),
- * since it does an integer division, x must be greater or equal to
- * 1000/CONFIG_SYS_CLOCK_TICKS_PER_SEC to get a non-zero value.
- * You may want to raise CONFIG_SYS_CLOCK_TICKS_PER_SEC depending on
- * your requirements.
- */
-#define SECONDS(x)	((x) * CONFIG_SYS_CLOCK_TICKS_PER_SEC)
-#define MSEC(x)		(SECONDS(x) / MSEC_PER_SEC)
-#define USEC(x)		(MSEC(x) / USEC_PER_MSEC)
-
 #ifdef __cplusplus
 }
 #endif

--- a/samples/drivers/gpio/src/main.c
+++ b/samples/drivers/gpio/src/main.c
@@ -104,6 +104,6 @@ void main(void)
 			toggle = 1;
 		}
 
-		k_sleep(SECONDS(2));
+		k_sleep(K_SECONDS(2));
 	}
 }

--- a/subsys/net/l2/ethernet/gptp/gptp_messages.c
+++ b/subsys/net/l2/ethernet/gptp/gptp_messages.c
@@ -14,7 +14,7 @@ LOG_MODULE_DECLARE(net_gptp, CONFIG_NET_GPTP_LOG_LEVEL);
 #include "gptp_md.h"
 #include "gptp_private.h"
 
-#define NET_BUF_TIMEOUT MSEC(100)
+#define NET_BUF_TIMEOUT K_MSEC(100)
 
 static struct net_if_timestamp_cb sync_timestamp_cb;
 static struct net_if_timestamp_cb pdelay_response_timestamp_cb;

--- a/tests/kernel/mem_protect/mem_protect/src/kobject.c
+++ b/tests/kernel/mem_protect/mem_protect/src/kobject.c
@@ -158,7 +158,7 @@ void test_kobject_revoke_access(void *p1, void *p2, void *p3)
 			0, K_INHERIT_PERMS | K_USER, K_NO_WAIT);
 
 
-	k_sem_take(&sync_sem, MSEC(100));
+	k_sem_take(&sync_sem, K_MSEC(100));
 	k_object_access_revoke(&kobject_sem, k_current_get());
 
 	k_thread_create(&kobject_test_4_tid,
@@ -215,7 +215,7 @@ void test_kobject_grant_access_kobj(void *p1, void *p2, void *p3)
 			0, K_INHERIT_PERMS | K_USER, K_NO_WAIT);
 
 
-	k_sem_take(&sync_sem, MSEC(100));
+	k_sem_take(&sync_sem, K_MSEC(100));
 
 	k_thread_create(&kobject_test_reuse_2_tid,
 			kobject_stack_2,
@@ -346,7 +346,7 @@ void test_kobject_access_all_grant(void *p1, void *p2, void *p3)
 			NULL, NULL, NULL,
 			0, K_USER, K_NO_WAIT);
 
-	k_sem_take(&sync_sem, MSEC(100));
+	k_sem_take(&sync_sem, K_MSEC(100));
 
 	k_thread_create(&kobject_test_reuse_2_tid,
 			kobject_stack_4,
@@ -405,7 +405,7 @@ void test_thread_has_residual_permissions(void *p1, void *p2, void *p3)
 			0, K_INHERIT_PERMS | K_USER, K_NO_WAIT);
 
 
-	k_sem_take(&sync_sem, MSEC(100));
+	k_sem_take(&sync_sem, K_MSEC(100));
 
 	k_thread_create(&kobject_test_9_tid,
 			kobject_stack_1,
@@ -956,4 +956,3 @@ void test_create_new_invalid_prio_thread_from_user(void *p1, void *p2, void *p3)
 	k_sem_take(&sync_sem, SYNC_SEM_TIMEOUT);
 
 }
-

--- a/tests/kernel/mem_protect/mem_protect/src/mem_domain.c
+++ b/tests/kernel/mem_protect/mem_protect/src/mem_domain.c
@@ -524,7 +524,7 @@ void test_mem_domain_remove_partitions(void *p1, void *p2, void *p3)
 			10, K_USER | K_INHERIT_PERMS, K_NO_WAIT);
 
 
-	k_sem_take(&sync_sem, MSEC(100));
+	k_sem_take(&sync_sem, K_MSEC(100));
 
 	k_mem_domain_remove_partition(&mem_domain_tc3_mem_domain,
 				      &mem_domain_tc3_part2_struct);

--- a/tests/kernel/pipe/pipe/src/test_pipe.c
+++ b/tests/kernel/pipe/pipe/src/test_pipe.c
@@ -40,8 +40,8 @@ ZTEST_BMEM u8_t rx_buffer[PIPE_SIZE];
 #define ALL_BYTES (sizeof(tx_buffer))
 
 #define RETURN_SUCCESS  (0)
-#define TIMEOUT_VAL (MSEC(10))
-#define TIMEOUT_200MSEC (MSEC(200))
+#define TIMEOUT_VAL (K_MSEC(10))
+#define TIMEOUT_200MSEC (K_MSEC(200))
 
 /* encompasing structs */
 struct pipe_sequence {

--- a/tests/kernel/semaphore/semaphore/src/main.c
+++ b/tests/kernel/semaphore/semaphore/src/main.c
@@ -14,7 +14,7 @@
 #define sem_give_from_isr(sema) irq_offload(isr_sem_give, sema)
 #define sem_take_from_isr(sema) irq_offload(isr_sem_take, sema)
 
-#define SEM_TIMEOUT (MSEC(100))
+#define SEM_TIMEOUT (K_MSEC(100))
 #define STACK_SIZE (1024 + CONFIG_TEST_EXTRA_STACKSIZE)
 #define TOTAL_THREADS_WAITING (5)
 
@@ -60,7 +60,7 @@ void sem_give_task(void *p1, void *p2, void *p3)
 
 void sem_take_timeout_forever_helper(void *p1, void *p2, void *p3)
 {
-	k_sleep(MSEC(100));
+	k_sleep(K_MSEC(100));
 	k_sem_give(&simple_sem);
 }
 
@@ -363,17 +363,17 @@ void test_sem_take_multiple(void)
 
 
 	/* time for those 3 threads to complete */
-	k_sleep(MSEC(20));
+	k_sleep(K_MSEC(20));
 
 	/* Let these threads proceed to take the multiple_sem */
 	k_sem_give(&high_prio_sem);
 	k_sem_give(&mid_prio_sem);
 	k_sem_give(&low_prio_sem);
-	k_sleep(MSEC(200));
+	k_sleep(K_MSEC(200));
 
 	/* enable the higher priority thread to run. */
 	k_sem_give(&multiple_thread_sem);
-	k_sleep(MSEC(200));
+	k_sleep(K_MSEC(200));
 	/* check which threads completed. */
 	signal_count = k_sem_count_get(&high_prio_sem);
 	zassert_true(signal_count == 1,
@@ -389,7 +389,7 @@ void test_sem_take_multiple(void)
 
 	/* enable the Medium priority thread to run. */
 	k_sem_give(&multiple_thread_sem);
-	k_sleep(MSEC(200));
+	k_sleep(K_MSEC(200));
 	/* check which threads completed. */
 	signal_count = k_sem_count_get(&high_prio_sem);
 	zassert_true(signal_count == 1,
@@ -405,7 +405,7 @@ void test_sem_take_multiple(void)
 
 	/* enable the low priority thread to run. */
 	k_sem_give(&multiple_thread_sem);
-	k_sleep(MSEC(200));
+	k_sleep(K_MSEC(200));
 	/* check which threads completed. */
 	signal_count = k_sem_count_get(&high_prio_sem);
 	zassert_true(signal_count == 1,
@@ -492,7 +492,7 @@ void test_sem_multiple_threads_wait(void)
 		}
 
 		/* giving time for the other threads to execute  */
-		k_sleep(MSEC(500));
+		k_sleep(K_MSEC(500));
 
 		/* Give the semaphores */
 		for (int i = 0; i < TOTAL_THREADS_WAITING; i++) {
@@ -500,7 +500,7 @@ void test_sem_multiple_threads_wait(void)
 		}
 
 		/* giving time for the other threads to execute  */
-		k_sleep(MSEC(500));
+		k_sleep(K_MSEC(500));
 
 		/* check if all the threads are done. */
 		for (int i = 0; i < TOTAL_THREADS_WAITING; i++) {
@@ -542,16 +542,16 @@ void test_sem_measure_timeouts(void)
 	/* With timeout of 1 sec */
 	start_ticks = k_uptime_get();
 
-	ret_value = k_sem_take(&simple_sem, SECONDS(1));
+	ret_value = k_sem_take(&simple_sem, K_SECONDS(1));
 
 	end_ticks = k_uptime_get();
 
 	zassert_true(ret_value == -EAGAIN,
 		     "k_sem_take failed when its shouldn't have");
 
-	zassert_true((end_ticks - start_ticks >= SECONDS(1)),
+	zassert_true((end_ticks - start_ticks >= K_SECONDS(1)),
 		     "time missmatch expected %d ,got %d\n",
-		     SECONDS(1), end_ticks - start_ticks);
+		     K_SECONDS(1), end_ticks - start_ticks);
 
 	/* With 0 as the timeout */
 	start_ticks = k_uptime_get();
@@ -606,16 +606,16 @@ void test_sem_measure_timeout_from_thread(void)
 	/* With timeout of 1 sec */
 	start_ticks = k_uptime_get();
 
-	ret_value = k_sem_take(&multiple_thread_sem, SECONDS(1));
+	ret_value = k_sem_take(&multiple_thread_sem, K_SECONDS(1));
 
 	end_ticks = k_uptime_get();
 
 	zassert_true(ret_value == 0,
 		     "k_sem_take failed when its shouldn't have");
 
-	zassert_true((end_ticks - start_ticks <= SECONDS(1)),
+	zassert_true((end_ticks - start_ticks <= K_SECONDS(1)),
 		     "time missmatch. expected less than%d ,got %d\n",
-		     SECONDS(1), end_ticks - start_ticks);
+		     K_SECONDS(1), end_ticks - start_ticks);
 
 }
 
@@ -661,14 +661,14 @@ void test_sem_multiple_take_and_timeouts(void)
 		k_thread_create(&multiple_tid[i],
 				multiple_stack[i], STACK_SIZE,
 				test_sem_multiple_take_and_timeouts_helper,
-				(void *)SECONDS(i + 1), NULL, NULL,
+				(void *)K_SECONDS(i + 1), NULL, NULL,
 				K_PRIO_PREEMPT(1), 0, K_NO_WAIT);
 	}
 
 	for (int i = 0; i < TOTAL_THREADS_WAITING; i++) {
 		k_pipe_get(&timeout_info_pipe, &timeout, sizeof(int),
 			   &bytes_read, sizeof(int), K_FOREVER);
-		zassert_true(timeout == SECONDS(i + 1),
+		zassert_true(timeout == K_SECONDS(i + 1),
 			     "timeout didn't occur properly");
 	}
 
@@ -716,11 +716,11 @@ void test_sem_multi_take_timeout_diff_sem(void)
 {
 	size_t bytes_read;
 	struct timeout_info seq_info[] = {
-		{ SECONDS(2), &simple_sem },
-		{ SECONDS(1), &multiple_thread_sem },
-		{ SECONDS(3), &simple_sem },
-		{ SECONDS(5), &multiple_thread_sem },
-		{ SECONDS(4), &simple_sem },
+		{ K_SECONDS(2), &simple_sem },
+		{ K_SECONDS(1), &multiple_thread_sem },
+		{ K_SECONDS(3), &simple_sem },
+		{ K_SECONDS(5), &multiple_thread_sem },
+		{ K_SECONDS(4), &simple_sem },
 	};
 
 	struct timeout_info retrieved_info;
@@ -751,7 +751,7 @@ void test_sem_multi_take_timeout_diff_sem(void)
 			   K_FOREVER);
 
 
-		zassert_true(retrieved_info.timeout == SECONDS(i + 1),
+		zassert_true(retrieved_info.timeout == K_SECONDS(i + 1),
 			     "timeout didn't occur properly");
 	}
 

--- a/tests/net/buf/src/main.c
+++ b/tests/net/buf/src/main.c
@@ -15,7 +15,7 @@
 
 #include <ztest.h>
 
-#define TEST_TIMEOUT SECONDS(1)
+#define TEST_TIMEOUT K_SECONDS(1)
 
 struct bt_data {
 	void *hci_sync;

